examples/docs/en-US/table.md

```diff
@@ -1385,6 +1385,7 @@ Customize table column so it can be integrated with other components.
 | row-dblclick | triggers when double clicking a row | row, event |
 | header-click | triggers when clicking a column header | column, event |
 | sort-change | triggers when Table's sorting changes | { column, prop, order } |
+| filter-change | triggers when column's filter condition changes. One object parameter which key is column's columnKey and key's value is filtered values of the column | filters | 
```

Comment:
triggers when column's filter condition changes. It's parameter is an object, whose keys are columns' `columnKey`s and values are arrays of filtered values of the column